src/main.py

```diff
@@ -27,6 +27,20 @@
 from utils import transform_keywords
 
 
+def filter_updates(update: Update) -> Optional[Update]:
```

Comment:
filters usually take in a collection and output a collection. You probably want a function that returns a `bool`. And you can make it a method on `Update`. E.g. `update.is_processable` `@property`.